selftests/unit/test_job.py

```diff
@@ -204,7 +205,10 @@ def test_job_run_account_time(self):
                   'show': ['none']}
         self.job = job.Job(config)
         self.job.setup()
+        # temporarily disable logging on console
+        logging.disable(logging.ERROR)
```

Comment:
One thing to consider here is that `show` is set to `['none']`, so in theory Avocado should not be producing any output. Maybe the best fix is to suppress it on the job class itself.

Reply:
That is an interesting point I have not noticed.

Reply:
Actually, I take it back, because of:

https://github.com/avocado-framework/avocado/blob/master/avocado/core/output.py#L46

It would require changing the semantics, so I guess your simpler approach is preferable at this point.